### ISSUE TITLE
Bounding boxes wrong locations for instances using "drawMode=bounds"

### ISSUE THREAD

**Issue by fabal** [PixarAnimationStudios/USD#1251]
### Description of Issue
It appears there's an issue in which bounding boxes are displayed in wrong locations since USD 20.02.

### Steps to Reproduce
1. Load this file in usdview

```
#usda 1.0

def Xform "ball_draw_mode" (
    instanceable = true
    kind = "component"
    # "Ball.geom.usd" comes from the Kitchen set
    prepend references = @./Ball.geom.usd@
)
{
    double3 xformOp:translate = (20, 0, 0)
    uniform token[] xformOpOrder = ["xformOp:translate", "xformOp:rotateXYZ", "xformOp:scale"]
    uniform token model:drawMode = "bounds"
}

def "ball_origin" (
    prepend references = @./Ball.geom.usd@
)
{}

def "ball_transformed" (
    prepend references = @./Ball.geom.usd@
)
{
    double3 xformOp:translate = (20, 0, 0)
    uniform token[] xformOpOrder = ["xformOp:translate", "xformOp:rotateXYZ", "xformOp:scale"]
}
```

### System Information (OS, Hardware)
linux

### Package Versions
USD 20.X

**Comment by jtran56** [comment]
Filed as internal issue #USD-6171.

**Comment by tcauchois** [comment]
Hi,

Just to double check, did this crash happen when you picked the object?  That's the only way I was able to repro it.  If so, the fix is in dev and 20.08 :).

Cheers,
Tom
